tests/nested/manual/core20-boot-config-update/task.yaml

```diff
@@ -21,9 +21,11 @@ prepare: |
       exit
   fi
 
-  "$TESTSTOOLS"/snaps-state repack-snapd-deb-into-snap snapd
+  # shellcheck source=tests/lib/prepare.sh
```

Comment:
I think in a following pr we should move out all the build/repack stuff from prepare.sh 
I'll work on that once this is merged